website/content/docs/job-specification/service.mdx

```diff
@@ -444,13 +491,46 @@ service {
   }
 
   check {
-    name     = "Postgres Check"
-    type     = "script"
-    command  = "/usr/local/bin/pg-tools"
-    args     = ["verify", "database", "prod", "up"]
-    interval = "5s"
+    name      = "Postgres Check"
+    type      = "script"
+    command   = "/usr/local/bin/pg-tools"
+    args      = ["verify", "database", "prod", "up"]
+    interval  = "5s"
+    timeout   = "2s"
+    on_update = "ignore_warnings"
+  }
+}
+```
+
+### Readiness and Liveness Checks
+
+Multiple checks for a service can be composed to create liveness and readiness
+checks by configuring [`on_update`][on_update] for the check.
+
+```hcl
+service {
+  check {
```

Comment:
Let's tell the user what we mean by "liveness and readiness", rather than relying on them bringing over concepts from k8s:

```suggestion
  # This is a liveness check that will be used to verify the service 
  # is up and serving traffic
  check {
```